h/services/group.py

```diff
@@ -21,11 +22,42 @@ def __init__(self, session, user_fetcher):
         self.session = session
         self.user_fetcher = user_fetcher
 
-    def fetch(self, pubid):
-        """Fetch a group by ``pubid``"""
+    def fetch(self, pubid_or_groupid):
+        """
+        Fetch a group, using either a groupid or a pubid
+
+        :param pubid_or_groupid: a string in either :mod:`~h.pubid` format
+                                 or as :attribute:`~h.models.group.Group.groupid`
```

Comment:
`:attribute:` doesn't exist in Sphinx, see <http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects> (you want `:attr:`)